smallest_multiple.js

```diff
@@ -8,7 +8,22 @@
  */
 module.exports = function( ceiling ) {
   // do work here
+  let numberToBeDivided = 1;
+  let smallestNumber;
+  let noRemainders = false;
+  
+  while (!noRemainders){
```

Comment:
In general, this whole while loop is pretty difficult to read. I would recommend getting rid of the `noRemainders` variable since you can just break out of the loop as soon as you've found a number that is evenly divisible by all the number below ceiling.